lib/cadet_web/controllers/user_controller.ex

```diff
@@ -54,6 +64,8 @@ defmodule CadetWeb.UserController do
               "Total maximum grade achievable based on submitted assessments." <>
                 "Only provided for 'Student'"
             )
+
+            xp(:integer, "Amount of xp. Only provided for 'Student'")
```

Comment:
"Amount of xp. Value will be 0 for non-students" or something along those lines.

Reply:
Done